core/src/trezor/messages.py

```
@@ -68,6 +68,7 @@ def __getattr__(name: str) -> Any:
     from trezor.enums import TezosContractType  # noqa: F401
     from trezor.enums import TonWalletVersion  # noqa: F401
     from trezor.enums import TonWorkChain  # noqa: F401
+    from trezor.enums import TronMessageType  # noqa: F401
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)